OSDOCS-16850 - CQA for OVNK-1 part 2 (#109553)
openshift/openshift-docs: modules/nw-ovn-kubernetes-examine-nb-database-contents-ref.adoc

[id="nw-ovn-kubernetes-examine-nb-database-contents-ref_{context}"]
= Command-line arguments for ovn-nbctl to examine northbound database contents

[role="_abstract"]

> The following table describes the command-line arguments that can be used with `ovn-nbctl` to examine the contents of the northbound database.

Reviewer: ```suggestion
The following table describes the command-line arguments that you can use with `ovn-nbctl` to examine the contents of the northbound database.
```
IMHO might as well apply some active voice since you are doing a CQA
